cpp/dolfin/fem/Form.cpp

```diff
@@ -74,6 +81,40 @@ int Form::original_coefficient_position(int i) const
   return _coefficients.original_position(i);
 }
 //-----------------------------------------------------------------------------
+void Form::set_constants(
+    std::map<std::string, std::shared_ptr<function::Constant>> constants)
+{
+  for (auto const& constant_in : constants)
```

Comment:
`constant_in` -> `constant`.

`const` should be not be necessary following `auto` if function argument is made `const`.